egs/librispeech/ASR/lstm_transducer_stateless/model.py

```diff
@@ -108,6 +109,11 @@ def forward(
             "sum" to sum the losses over all utterances in the batch.
             "none" to return the loss in a 1-D tensor for each utterance
             in the batch.
+          delay_penalty:
+            A constant value used to penalize symbol delay, to encourage
+            streaming models to emit symbols earlier.
+            See https://github.com/k2-fsa/k2/issues/955 and
+            https://arxiv.org/pdf/2211.00490.pdf for more details.
```

Comment:
Great paper ! One comment I notice, Fig 3a and 4.a seems to have the WER number wrong (since it is suppose to be `test-clean` which are in the 3-4 %), is that correct ?

Reply:
Thanks. The presented results in figures 3, 4, 5 are averaged over the test-clean and test-other subsets for comparison.